### ISSUE TITLE
Add post install script to interactively define environment variables

### ISSUE THREAD

**Issue by vidurb** [miranj/craft#6]
I think this would be a good value add, but I don't think I'm going to see agreement on that. Putting it out there anyway.

**Comment by rungta** [comment]
`./craft setup` does this. Are you talking about extending this to include all environment variables?

**Comment by vidurb** [comment]
Yes, I'm talking about the environment values that would normally be defined in `.env`.
